walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -209,6 +208,67 @@ public void fetchJob_NormalInput_success() throws ExecutionException, Interrupte
         assertEquals(job, actualJob);
     }
 
+    @Test
+    public void hasJob_normalInput_true() throws ExecutionException, InterruptedException {
+        // Arrange.
+        Job job = new Job();
+        Future<DocumentReference> addedJobFuture = firestore.collection(TEST_JOB_COLLECTION).add(job);
+
+        DocumentReference documentReference = addedJobFuture.get();
+        // Asynchronously retrieve the document.
+        ApiFuture<DocumentSnapshot> future = documentReference.get();
+
+        // future.get() blocks on response.
+        DocumentSnapshot document = future.get();
+        String jobId = document.getId();
+
+        try {
+            // Act.
+            Future<Boolean> result = JobsDatabase.hasJob(jobId);
+
+            // Assert.
+            assertTrue(result.get());
+        } catch (IllegalArgumentException e) {
```

Comment:
you should not catch this here since it's not part of your test. just add IllegalArgumentException to your test signature.